lib/syskit/models/deployment_group.rb

```diff
@@ -459,10 +464,11 @@ def use_deployment(
                         process_managers.default_run_options(model)
                     )
 
+                    read_only_model = read_only ? [name] : []
```

Comment:
This is wrong. `read_only` is meant to be a list of deployed task names, `name` is ... I don't know what actually.

Is there a test about this ?

Reply:
Yeah, the test is on `test_deployment_group`.